app/src/main/java/com/support/android/designlibdemo/activities/SignUpActivity.java

```diff
@@ -1,29 +1,62 @@
 package com.support.android.designlibdemo.activities;
```

Comment:
We already discussed that 3 times already. We are not planning to enable picture capture functionality for sign up flow.